composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=7.2.0"
```

Comment:
why 7.2 and not 7.0 since that's all this package needs?

Reply:
Argument widening is needed to avoid a breaking change.

Reply:
This will be tagged as a minor version. We need PHP 7.2 because we need support of the parameter type widening (https://blog.kelunik.com/2018/01/23/parameter-type-widening.html) (since we are adding a more specific type in the interface and implementations will not have the type if they target 1.0)

Reply:
> Argument widening is needed to avoid a breaking change.

indeed! And we did the same on Symfony for the same reason... sorry for the noise :)